js/src/volume.js

```diff
@@ -310,6 +308,25 @@ var ScatterView = widgets.WidgetView.extend( {
                 this.previous_values["vz"] = this.model.get("vz")[pindex]
                 this.attributes_changed["vz"] =["vz"]
               }
+
+              if (this.model.get("color") ) {
+                  color = this.model.get("color")
+                  if ( typeof color == "string" || typeof color[0] == "string" || typeof color[0][0] == "string") {
+                      if (!(typeof color == "string" || typeof color[0] == "string")){
```

Comment:
Could you make this a bit simpler, sth like:
```javascript
 if ( ... 0 or 1d check ) {
 // don't do anything, we don't have a sequence, 
}
else if ( .. 2d check ...) {
  // we have a sequence
...
}
```
I think this is more easy to read. 
Please be consistent with whitespace, no whitespace after `if(`,